debugger/ptraceLib/oe_ptrace.c

```diff
@@ -306,7 +309,7 @@ int64_t ptrace(oe_ptrace_request_type __request, ...)
 
     // If the request should be handled by the customized handler, calls
     // customer handler.
-    for (uint32_t i = 0; i < OE_COUNTOF(g_request_handlers); i++)
+    for (uint32_t i = 0; i < COUNTOF(g_request_handlers); i++)
```

Comment:
This change is good, but can we rename the file to not have oe_ in the filename now?

Reply:
Done. Additionally, changed `oe_` prefix to `sgx_` everywhere.